imageUtil.ml

```diff
@@ -216,6 +216,27 @@ let print_byte v =
  *)
 let rec ones i = if i == 1 then 1 else ((ones (i-1)) lsl 1) lor 1 ;;
 
+(*
+ * Converts a string of size 1 into an integer.
+ * Arguments:
+ *   - s : the string.
+ * Returns an integer.
+ *)
+let int_of_str1 s =
+  assert (String.length s >= 1);
```

Comment:
maybe we can throw `Invalid_argument` instead of the `assert` for these?

Reply:
I don't see what we gain from this. `Invalid_argument` here, `Assertion_failure` there, it makes little difference, apart from the syntax for `assert` being cleaner.

Reply:
If we use both, the user has to catch both to be able to handle errors gracefully.
That means we need to document both in the `mli`, which leads to bigger docstrings and more things to worry about for the user.

Reply:
FWIW: assertions can be disabled (see the `-noassert` option to `ocaml`) - in my experience this means that assertions should not be relied upon, but instead an error should be returned (or an exception raised).

Reply:
Oh, I didn't know that, that's a good point, Hannes.